src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task08.java

```diff
@@ -38,8 +38,51 @@ public static void main(String[] args) {
      * @return итоговый урон по данному типу моба
      */
     static int getTotalDamage(int damage, String typeMob, boolean hasHolyAttribute) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        if (typeMob == "UNDEAD") {
+            if (hasHolyAttribute == true) {
+                damage = (int) (damage * 1.5);
+            }
+            return damage;
+        }
+
+        if (typeMob == "ZOMBIE"){
+            if (hasHolyAttribute == true) {
+                damage = (int) (damage * 1.5);
+            }
+            return damage;
+        }
+
+        if (typeMob == "SAINT") {
+            if (hasHolyAttribute == true) {
+                damage = damage / 2;
+            }
+            return damage;
+        }
+
+
+        if(typeMob == "ANIMAL"){
+            if (hasHolyAttribute == true) {
+                return damage;
+            }
+        }
+
+        if(typeMob == "HUMANOID"){
+            if (hasHolyAttribute == true) {
+                return damage;
+            }
+        }
+
+        if(typeMob == "PLANT"){
+            if (hasHolyAttribute == true) {
+                return damage;
+            }
+        }
+
+        if(typeMob == "GHOST"){
+            if (hasHolyAttribute == true) {
+                return damage;
+            }
+        }
+        return damage;
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"